### PR TITLE
hwdb: add touchpad for Lenovo Thinkpad T430

### PR DESCRIPTION
**Pull request by sammynx**
Added correct range for the Lenovo Thinkpad T430 touchpad.

### CODE REVIEW

File: hwdb/60-evdev.hwdb [hunk 1]
```diff
@@ -253,6 +253,13 @@ evdev:name:SynPS/2 Synaptics TouchPad:dmi:*svnLENOVO*:pvrThinkPad??40?:*
  EVDEV_ABS_35=::41
  EVDEV_ABS_36=::37
 
+# LENOVO Thinkpad T430
```
Comment (AsciiWolf) [inline hwdb/60-evdev.hwdb]: Minor typo: LENOVO -> Lenovo

Comment (AsciiWolf) [reply hwdb/60-evdev.hwdb]: Thinkpad -> ThinkPad

Comment (sammynx) [reply hwdb/60-evdev.hwdb]: changed the typos

File: hwdb/60-evdev.hwdb [hunk 2]
```diff
@@ -253,6 +253,13 @@ evdev:name:SynPS/2 Synaptics TouchPad:dmi:*svnLENOVO*:pvrThinkPad??40?:*
  EVDEV_ABS_35=::41
  EVDEV_ABS_36=::37
 
+# Lenovo ThinkPad T430
```
Comment (martinpitt) [inline hwdb/60-evdev.hwdb:256]: I cannot spot what on earth is wrong with this comment, but `hwdb/parse_hwdb.py` fails with
```
Cannot parse ../hwdb/60-evdev.hwdb: Expected stringEnd (at char 7102), (line:256, col:1)
```

Is there some funny non-standard space there or something similar?

**Comment by martinpitt** [comment]
Please update the commit message to "hwdb: add touchpad for Lenovo Thinkpad T430" (i. e. prefix with "hwdb:" and mention the precise change, not just "update"). Thanks!

**Comment by poettering** [comment]
The CI is failing. Also, could you please merge the two patches into one? (git rebase -i allows doing that)

**Comment by poettering** [comment]
also, cc @whot

File: hwdb/60-evdev.hwdb (hunk 2 above)
Comment (sammynx) [reply hwdb/60-evdev.hwdb:256]: Don't know about funny spaces, but I typed it in by hand now to get rid of any funny characters.

File: hwdb/60-evdev.hwdb [hunk 3]
```diff
@@ -253,6 +253,13 @@ evdev:name:SynPS/2 Synaptics TouchPad:dmi:*svnLENOVO*:pvrThinkPad??40?:*
...
+evdev:name:SynPS/2 Synaptics TouchPad:dmi:*svnLENOVO*:pvrThinkpadT430*
```
Comment (AsciiWolf) [inline hwdb/60-evdev.hwdb]: Why "Thinkpad" instead of "ThinkPad"?

File: hwdb/60-evdev.hwdb (hunk 2 above)
Comment (sammynx) [reply hwdb/60-evdev.hwdb:256]: found some non printing character and removed it. File parses now.

**Comment by whot** [comment]
I'll need you to squash the commits together into logical changes, Please ping me when you've done this and I can review it then. Thanks

File: hwdb/60-evdev.hwdb [hunk 4]
```diff
@@ -45,6 +45,7 @@
 
 #  Macbook2,1 (late 2006), single-button touchpad
 evdev:input:b0003v05ACp021B*
+
```
Comment (martinpitt) [inline hwdb/60-evdev.hwdb]: These empty lines break the format, as all consecutive "evdev:" lines apply to the same indented block of `EVDEV_*` properties set below. Please remove them again.

File: hwdb/60-evdev.hwdb (hunk 2 above)
Comment (martinpitt) [reply hwdb/60-evdev.hwdb:256]: No, tests still fail. Please run `make check` or `parse_hwdb.py` manually to verify.

**Comment by sammynx** [comment]
squashed and merged my commits into my master and made a new pull request #5066

**Comment by keszybz** [comment]
OK, tests pass and it seems all comments have been adressed.

**Comment by sammynx** [comment]
Thanks

**Pull request merged**